W-14250766-added 4.5 example. Removed unsupported sample (#2566)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/deploy-to-cloudhub.adoc

If not set, by default this value is set to +https://anypoint.mulesoft.com+. | No
| `muleVersion` | The Mule runtime engine version to run in your CloudHub instance.  +
Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
Example value: `4.3.0` +

> Starting with Mule 4.5, deployments to CloudHub require Major.Minor version which deploys the latest version of Mule runtime. 

Reviewer: Sorry! I missed one comma. It should read:
"Starting with Mule 4.5, deployments to CloudHub require Major.Minor version, which deploys the latest version of Mule runtime".

> Starting with Mule 4.5, deployments to CloudHub require Major.Minor version which deploys the latest version of Mule runtime. 

Reviewer 2: ```suggestion
Starting with Mule 4.5, deployments to CloudHub require Major.Minor version, which deploys the latest version of Mule runtime. 
```

Reviewer 3: I think we should put something like: latest patch version for the given minor

> Starting in 4.5, only the Major.Minor will be required for deployments to CloudHub. This will deploy the latest version of the Mule Runtime. 

Reviewer 2: ```suggestion
Starting with Mule 4.5, deployments to CloudHub require Major.Minor version which deploys the latest version of Mule runtime. 
```

> Starting in 4.5, only the Major.Minor will be required for deployments to CloudHub. This will deploy the latest version of the Mule Runtime. 

Reviewer: According to our style guide, "only" is hardly needed and we should choose present instead of future tense.
